runhouse/rns/rns_client.py

```diff
@@ -30,7 +30,7 @@ class ResourceStatusData(BaseModel):
     system_cpu_usage: float
     system_memory_usage: Dict[str, Any]
     system_disk_usage: Dict[str, Any]
-    env_servlet_processes: Dict[str, Dict[str, Any]]
+    env_servlet_processes: Optional[Dict[str, Dict[str, Any]]] = {}
```

Comment:
Could be an empty dictionary because we start checking and sending status to den once the cluster servlet is up. This covers an edge case where there not runhouse envs on the cluster yet => there is no` env_servlet_processes`. (can happen in the first checks, before the default env and/or other envs are set).